### ISSUE TITLE
Incorrect information regarding versioning on `Deleting multiple objects`

### ISSUE THREAD

**Issue by ermiyaeskandary** [awsdocs/amazon-s3-userguide#68]
Hello,

On the ['Deleting multiple objects'](https://docs.aws.amazon.com/AmazonS3/latest/userguide/delete-multiple-objects.html) page, the below warning is given:

> To delete an object in a versioning-enabled bucket with versioning Off, Amazon S3 creates a delete marker. To undo the delete action, delete this delete marker. To confirm this action, type delete.
> 
> To delete an object version in a versioning-enabled bucket with versioning On, Amazon S3 will permanently delete the object version. To confirm this action, type permanently delete.

This is **incorrect** & is the other way round.

It should be:

> To delete an object in a versioning-enabled bucket with versioning Off, Amazon S3 will permanently delete the object version. To confirm this action, type permanently delete.
> 
> To delete an object version in a versioning-enabled bucket with versioning On, Amazon S3 creates a delete marker. To undo the delete action, delete this delete marker. To confirm this action, type delete.

**Comment by aws-dianakm** [comment]
Confirmed in the S3 console that the feedback is valid. To make the wording less confusing, I've updated it to the following:
"To delete an object in a bucket with bucket versioning disabled or suspended, Amazon S3 creates a delete marker. To undo the delete action, delete this delete marker. To confirm this action, type permanently delete.

To delete an object version in a bucket with bucket versioning enabled, Amazon S3 will permanently delete the object version. To confirm this action, type delete."

Once I've verified the changes in our build, I'll close out this request.
